src/App.tsx

```diff
@@ -1,14 +1,42 @@
-/* eslint-disable max-len */
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 import 'bulma/css/bulma.css';
 import '@fortawesome/fontawesome-free/css/all.css';
 
 import { TodoList } from './components/TodoList';
 import { TodoFilter } from './components/TodoFilter';
 import { TodoModal } from './components/TodoModal';
 import { Loader } from './components/Loader';
+import { getTodos } from './api';
+import { useAppDispatch, useAppSelector } from './app/hooks';
+import { actions as TodosActions } from './features/todos';
+import { getFilteredTodos } from './utils/helpers/filterTodos';
 
 export const App: React.FC = () => {
+  const todos = useAppSelector(state => state.todos);
+  const selectedTodo = useAppSelector(state => state.currentTodo);
+  const filter = useAppSelector(state => state.filter);
+  const dispatchTodos = useAppDispatch();
+
+  const [isLoaded, setIsLoaded] = useState(false);
+
+  useEffect(() => {
+    const fetchTodo = async () => {
```

Comment:
could you please also read about redux-thunk
https://github.com/reduxjs/redux-thunk

Reply:
I don't understand, 
![image](https://github.com/mate-academy/react_redux-list-of-todos/assets/104682987/95c2c2fe-a09d-4451-bfac-36c120acc7d6)